javascript-bank/account.js

```diff
@@ -0,0 +1,58 @@
+/* exported Account */
+
+function Account(number, holder){
+  this.number = number;
+  this.holder = holder;
+  this.transactions = [];
+}
+
+Account.prototype.number = function () {
+  return this.number;
+}
+
+Account.prototype.holder = function () {
+  return this.holder;
+}
+
+
+Account.prototype.deposit = function (amount) {
+
+  var account = new Transaction();
+
+  if ((amount === 'fake money') || (amount === NaN) || (amount <= 0) || (amount % 1 !== 0)) {
+    return false;
```

Comment:
The `deposit` and `withdraw` methods should return `false` for _all_ non-numeric `amount`s, not just `NaN` and the string `'fake money'`.